### DISCUSSION TITLE
--watch command doesn't rebuild @import files automatically

### DISCUSSION THREAD

**Discussion by erlangga0** [tailwindlabs/tailwindcss#9614] [Help] [answered]
Hi, I found an issue after updating tailwind from 3.1.8 to 3.2.0
The --watch command no longer rebuilds output file after I save an imported css file (using @import).

Here is the package.json:
`"scripts": {
    "watch": "npx tailwindcss -i ./src/css/main.css -o ./public/css/style.css --postcss --watch"
  }`
....
`"devDependencies": {
    "autoprefixer": "^10.4.12",
    "postcss": "^8.4.18",
    "postcss-import": "^15.0.0",
    "tailwindcss": "^3.2.0"
  }`
  
And the main.css:
`@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
@import "./custom.css";`

So whenever I save **custom.css**, tailwind doesn't rebuild  **./public/css/style.css**, unlike the previous version (3.1.8)

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your bug report!
Much appreciated! 🙏

This should be fixed by #9617, and will be available in the next release.

You can already try it by using the insiders build `npm install tailwindcss@insiders`.
